packages/adblocker/src/filters/cosmetic.ts

```diff
@@ -786,23 +786,28 @@ export default class CosmeticFilter implements IFilter {
               inArgument = true;
             }
           }
-          if (char === ',') {
-            parts.push(selector.slice(lastComaIndex + 1, index).trim());
-            lastComaIndex = index;
-            inArgument = false;
-          }
         }
       }
 
-      lastCharIsBackslash = char === '\\';
```

Comment:
Preserving this line and putting into "else" block of `lastCharIsBackslash === false` state might be cleaner.